### DISCUSSION TITLE
Few Concerns on MeshCentral

### DISCUSSION THREAD

**Discussion by mdshoaibumer** [Ylianst/MeshCentral#3682] [Q&A]
Dear @Ylianst ,

I have few things to enquire, kindly suggest on this :

1: Remote Disconnect Pop Text not showing:- If admin / viewer disconnects the session , the pop-up text or Notification should be triggered on End User Screen.
 
2. End User not able to stop file transfer in the middle of process:- Ones admin/Viewer Start File Transfer mode , the end user must have rights to stop or disable the file transfer .
 
3. User require session disconnect rights :- End User must have rights to disconnect the session in between.

**Comment by nccerhostmaster** [comment]
2 and 3 do not seem useful to me. Don't need even more ways for remote users to make mistakes.

**Reply by VINISHVV** [reply]
For point 1 and 3 enabld user consent with notification bar

**Comment by Ylianst** [comment]
These are all good points. They need to be new issues in the issues list. It looks like your asking for the privacy bar to show for Desktop/Terminal/Files, not just Desktop. I think this is a good idea. Closing that bar should close all sessions.

I will note that is your download MeshCentral Assistant and use it just as an agent monitoring tool, you should see a log of what the agent is doing and who is doing it. If I recall correctly, you can disconnect sessions using that tool, but this is not commonly installed with the MeshCentral agent.
